notifyQueue.js

```diff
@@ -2,15 +2,37 @@ const Queue = require('bull');
 const { setQueues } = require('bull-board');
 const { UI } = require('bull-board');
 
-const notifyQueue = new Queue('sendMail',
-	{
-		redis:{
-			host:'127.0.0.1',
-			port: 6379
+
+const redisUri = process.env.REDIS_URI || 'x-notify-redis';
+const redisPort = process.env.REDIS_PORT || '6379';
+const redisSentinel1Uri = process.env.REDIS_SENTINEL_1_URI || '127.0.0.1';
+const redisSentinel1Port = process.env.REDIS_SENTINEL_1_PORT || '26379';
+const redisSentinel2Uri = process.env.REDIS_SENTINEL_2_URI || '127.0.0.1';
+const redisSentinel2Port = process.env.REDIS_SENTINEL_2_PORT || '26379';
+const redisMasterName = process.env.REDIS_MASTER_NAME || 'x-notify-master';
+
+
+let redisConf = {};
+if (process.env.NODE_ENV === 'production') {
```

Comment:
According to documentation (line 77 in setup.md), the value need to be set to:  `prod`